features/facebookImport/src/model/importers/comments-importer.js

```diff
@@ -0,0 +1,23 @@
+import DirectKeyDataImporter from "./direct-key-data-importer";
+
+export const COMMENTS_FILE_PATH = "comments_and_reactions/comments.json";
+export const COMMENTS_DATA_KEY = "comments_v2";
+export const COMMENTS_STORAGE_KEY = "comments";
+
+export default class CommentsImporter extends DirectKeyDataImporter {
+    constructor() {
+        super(COMMENTS_FILE_PATH, COMMENTS_DATA_KEY, COMMENTS_STORAGE_KEY);
+    }
+
+    _extractDataFromComment(commentData) {
+        return {
+            timestamp: commentData.timestamp,
```

Comment:
Could this have common code with other places where we want to extract the timestamp? We might possibly need only the name of the data key (and maybe the key used for timestamp)

Reply:
Could be. At the moment we do the same when importing the posts, but the importer there has different logic as it needs to handle a collection of files. We can generalise this when get a second similar import.